torch/testing/_internal/common_nn.py

```diff
@@ -275,8 +295,11 @@ def _rand_tensor_non_equal(*size):
 
 def wrap_functional(fn, **kwargs):
     class FunctionalModule(nn.Module):
+        def __init__(self):
```

Comment:
Why did the implementation of `wrap_functional` change? Is there a reason why the old one doesn't work?

Reply:
Thanks for the catch - it was an artifact of an old implementation to get name of the functional from `.fn`, but it's not needed anymore :D